docs/src/main/sphinx/sql/merge.rst

```diff
@@ -0,0 +1,72 @@
+=====
+MERGE
+=====
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    MERGE INTO target_table [ [ AS ]  target_alias ]
+        USING
+            { source_table | query } [ [ AS ] source_alias ]
+        ON search_condition
+        {
+            WHEN MATCHED [ AND condition ]
+                THEN (DELETE | UPDATE SET column = expression, ...))
+         |
+            WHEN NOT MATCHED [ AND condition ]
+                THEN INSERT [ column_list ] VALUES (expression [, expression ]*))+
+        }
```

Comment:
I can see `))+` and `}` in italic.

Reply:
There were multiple problems with the syntax diagram, and @electrum also made some changes.  It's better now.